src/commands/gitHub/connectToGitHub/connectToGitHub.ts

```diff
@@ -48,7 +48,11 @@ export async function connectToGitHub(context: ITreeItemPickerContext & Partial<
         new GitHubOrgListStep(),
         new GitHubRepositoryListStep(),
         new GitHubBranchListStep(),
-        new DockerfileLocationInputStep(),
+        new GitHubRepositoryFileSelectStep(
+            'Dockerfile',  // fileName
```

Comment:
The action only works properly when this case-sensitive version of the `Dockerfile` is provided.